dallinger/experiment_server/experiment_server.py

```diff
@@ -572,11 +573,16 @@ def create_participant(worker_id, hit_id, assignment_id, mode):
         'participant': participant.__json__()
     }
 
+    exp = Experiment(session)
+
+    # Ping back to the recruiter that one of their participants has joined:
+    recruiter = Recruiter.for_experiment(exp)
+    recruiter.notify_recruited(participant, exp)
```

Comment:
Not sure. The thing that is slow: looking for a qualification name that either does not exist, or has only just been added, and so the name hasn't been indexed yet on MTurk (this is what makes tests slow). In practice, this would bite the first worker to join an experiment if the experiment author/admin had not already created the Qualification. Maybe we wait and see what the actual experience is like?